services/ui-src/src/Routes.test.tsx

```diff
@@ -0,0 +1,264 @@
+import { render, screen, act } from "@testing-library/react";
+import { renderHook } from "@testing-library/react";
```

Comment:
Let's combine these imports:

```suggestion
import { act, render, renderHook, screen } from "@testing-library/react";
```

Reply:
omg good catch